source/whats-new.txt

```diff
@@ -42,6 +43,20 @@ Learn what's new in:
 * :ref:`Version 1.1 <version-1.1>`
 * :ref:`Version 1.0 <version-1.0>`
 
+.. _version-1.15:
+
+What's New in 1.15
+------------------
+
+The 1.15 {+driver-short+} release includes the following improvements and fixes:
+
+- The driver mitigates connection churn by attempting to reuse a connection
+  after an operation times out. The driver waits for up to one second to check 
+  if the connection can be reused before closing it.
```

Comment:
It would be helpful to note somewhere that the "mitigating connection churn" feature is only enabled when a Client-wide timeout is set (either by [SetTimeout](https://pkg.go.dev/go.mongodb.org/mongo-driver/mongo/options#ClientOptions.SetTimeout) or the `timeoutMS` connection string option).